Dockerfile

```diff
@@ -25,4 +27,13 @@ WORKDIR /code
 RUN pip3 install --no-cache-dir -r requirements.txt
 
 # build assets
-RUN npm ci && npm run build
+RUN cd taccsite_custom\
+    # install node packages (`yarn 1.x` equivalent of `npm ci`)
+    && rm -rf node_modules && yarn install --frozen-lockfile\
```

Comment:
What's the reasoning for using yarn instead of npm?

Reply:
I have no good reasoning.

<details>
<summary>Why I Switch?</summary>

I was trying it out, then I converted to it when I forgot I was just learning. I had realized what I'd done without considering inconsistency, then forgot later. Thank you for bringing it up.

</details>
<details>
<summary>Why Not Switch?</summary>

1. NPM seems to have *fixed* the problems Yarn came to solve[^proof].
2. [Yarn does not support `npm_config`](https://github.com/yarnpkg/yarn/issues/4581), a feature that reduces complexity when scripting.
3. Yarn is [giddy about a 2.0 release](https://dev.to/arcanis/introducing-yarn-2-4eh1), while NPM is far more *mature*.
    - And Yarn 2.0 is 2nd-class citizen until Node [Corepack](https://nodejs.org/dist/latest/docs/api/corepack.html#corepack) is stable.

[^proof]: I did not save the articles I read on NPM catching up to Yarn.

</details>
<details><summary>Why To Switch?</summary>

1. Yarn is, or at least _feels_[^speed], *faster* (maybe because of [parallelism](https://www.section.io/engineering-education/npm-vs-yarn-which-one-to-choose/#the-speed), and because of usually shorter commands).
2. Yarn's lock file is *easier* to read than NPM's.
3. They can be used together (like I have) to usually get the best of both worlds.

[^speed]: After days of use, it really does feel so much faster. I have often run npm and yarn after each other for the same task using saved lock file, using empty `node_modules`, and to drop or fresh install a dependnecy—yarn pwns.

</details>

(@taoteg, you may have interest in the above details.)

Reply:
I have removed Yarn.

<details><summary>Why?</summary>

- It is not required for this change.
- We can evaluate it later.
- If we switch it should be wholesale.

</details>